backend/src/appointment/routes/caldav.py

```diff
@@ -86,21 +87,11 @@ def caldav_autodiscover_auth(
         lookup_url = Tools.well_known_caldav_lookup(connection.url)
 
     # If we have a lookup_url then apply it
-    if lookup_url:
+    if lookup_url and 'http' not in lookup_url:
+        connection.url = urllib.parse.urljoin(connection.url, lookup_url)
```

Comment:
Join the absolute and relative urls. 

Ex/ lookup_url = '/cal/dav/' and connection.url = 'https://example.org/' = 'https://example.org/cal/dav'